assets/js/script.js

```diff
@@ -0,0 +1,178 @@
+    document.getElementsByClassName('play_bgsound')[0].volume = 0.8
+    document.getElementsByClassName('play_bgsound')[0].play()
+  }
+
+  function hitOver() {
+    document.getElementsByClassName('hit')[0].play()
+  }
+  // audio
+
+  spop()
+
+  // start instruction popup
+  $startBtn = $('.start')
+  $startBtn.on('click', () => {
+    document.getElementsByClassName('startpop')[0].pause()
+    stopKeyDown = true;
+    $('.start_pop').hide()
+    playBgSound ()
+    speed = 20
+  })
+
+	$container.on('mousedown', function () {
+    start()
+    birdFlap ()
+    flapsound()
+	})
+
+	$(this).on('keydown', function (e) {
+    if (stopKeyDown) {
+      start()
+      birdFlap ()
+      flapsound()
+      if (e.keyCode === 32) {
+        if (gameState > 1) return
+        if (gameState === 2) gameState = 1
+      }
+    }
+  })
+
+
+  var polePosInterval = setInterval(poleMove, 50)
+
+  var birdPosInterval = setInterval(function () {
+    collisionDetection()
+  }, 50)
+
+  // collision session
+
+  // collision with pole
+  function collisionDetection () {
+    if ($bird.position().left < poleTop.position().left + poleTop.width() && $bird.position().left + $bird.width() > poleTop.position().left && $bird.position().top < poleTop.position().top + poleTop.height() && $bird.height() + $bird.position().top > poleTop.position().top) {
+      gameEnd()
+    }
+    if ($bird.position().left < poleBottom.position().left + poleBottom.width() && $bird.position().left + $bird.width() > poleBottom.position().left && $bird.position().top < poleBottom.position().top + poleBottom.height() && $bird.height() + $bird.position().top > poleBottom.position().top) {
+      gameEnd()
+    }
+  }
+
+  // collision with top and bottom container
+  function birdPos () {
+    if (parseInt($bird.css('top')) <= 0 || parseInt($bird.css('top')) > $('.container').height() - $('.bird').width()) {
+      gameEnd()
+    }
+  }
+  // End collision session
+
+  function poleMove () {
+    var cPolePos = parseInt(pole.css('right'))
+    // update the score when the poles have passed the $bird successfully
+    if (cPolePos > conWidth - $birdLeft) {
+      if (scoreAdd === false) {
+        score.text(parseInt(score.text()) + 1)
+        scoreAdd = true
+      }
+    }
+    // check whether the poles went out of the $container
+    if (cPolePos > conWidth) {
+      var newHeight = parseInt(Math.floor(Math.random() * 300))
+      // change the pole's height
+      poleTop.css('height', pole_initial_height + newHeight)
+      poleBottom.css('height', pole_initial_height - newHeight)
+      //speed increase +1 every time a new pole enter
+      speed = speed + 1;
+      scoreAdd = false
+      // move pole back to right
+      cPolePos = polePos
+    }
+    // move the poles
+    pole.css('right', cPolePos + speed)
+  }
+
+  function birdFlap () {
+    if (gameState === 1 || gameState === 2) {
+      $bird.css('transform', 'rotate(-20deg)')
+      $bird.stop().animate({
```

Comment:
Why do you use animate method instead of doing this with [`css animation`](https://developer.mozilla.org/en-US/docs/Web/CSS/animation) property instead?